src/client/components/UserHeader.js

```diff
@@ -87,11 +87,6 @@ const UserHeader = ({
           <div className="UserHeader__handle-rank-container">
             <div className="UserHeader__row UserHeader__handle">
               @{handle}
-              {displayFollowsYou && (
-                <span className="UserHeader--follows-you-indicator">
-                  {intl.formatMessage({ id: 'follows_you', defaultMessage: 'follows you' })}
```

Comment:
Can you use FormattedMessage instead of formatMessage?
If you changed text to Follows you, you should update defaultMessages as well.

Reply:
okay.